manifests/g/GaëtanSencie/Comptabilitéannuelle/1.7.0.0/GaëtanSencie.Comptabilitéannuelle.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GaëtanSencie.Comptabilitéannuelle
+PackageVersion: 1.7.0.0
+InstallerLocale: fr-BE
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /passive
+  Interactive: "true"
+  InstallLocation: C:\Program Files\Comptabilité annuelle

```

Comment:
A location should not be hardcoded. If the location is needed, then `InstallLocationRequired: true` should be specified and the install location should be the switch needed to set the install location